security/cert_manager_operator/cert-manager-creating-certificate.adoc

```diff
@@ -0,0 +1,26 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cert-manager-creating-certificate"]
+= Configuring certificates with an issuer
+include::_attributes/common-attributes.adoc[]
+:context: cert-manager-creating-certificate
+
+toc::[]
+
+You can manage certificates, including tasks like renewal and issuance of certificates, by using {cert-manager-operator}. {cert-manager-operator} can manage certificates for workloads running on the cluster, and the components interacting externally to the cluster.
```

Comment:
`{cert-manager-operator}. {cert-manager-operator}` is repeated at the end of the sentence and right after in next. You might want to change that, not sure.